c2corg_api/models/user.py

```diff
@@ -103,6 +103,10 @@ class User(Base):
     feed_filter_activities = Column(
         ArrayOfEnum(enums.activity_type), nullable=False, server_default='{}')
 
+    # only show updates from followed users in the homepage feed
+    feed_followed_only = Column(
```

Comment:
OK, I remember that we have discussed the possibility to only show followed contents in the feed.
I guess we have to add a flag into the user prefs UI and to support it in the API service (see https://github.com/c2corg/v6_ui/pull/676 and https://github.com/c2corg/v6_api/issues/402#issuecomment-252204075)